src/jsifier.mjs

```diff
@@ -286,15 +286,22 @@ ${argConversions}
     if (LIBRARY_DEBUG && !isJsOnlySymbol(symbol)) {
       snippet = modifyJSFunction(
         snippet,
-        (args, body, async) => `\
+        (args, body, async, oneliner) => {
+          var dobody;
```

Comment:
lgtm but I'm not sure what the word `dobody` means?

Reply:
Renamed.